### PR TITLE
feature/chart-minimap

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
### Summary
Displaying minimap for a chart. It provides:
1. Zoom;
2. Moving through data in zoom;

### Screenshots
image

### CODE REVIEW

**Comment by Partysun** [comment]
Brush is shifted from 0 x point of the chart
image

**Comment by Partysun** [comment]
image
What is happened here? @DmitriVanGuard

**Comment by DmitriVanGuard** [comment]
> image
> What is happened here? @DmitriVanGuard

@Partysun https://www.figma.com/file/RPLINi2nrD30hQjcANws3CxH/Sanbase?node-id=1192%3A236
image

File: src/ducks/SANCharts/Charts.js [hunk 1]
```diff
@@ -110,6 +115,7 @@ class Charts extends React.Component {
     if (!(this.metricRef.current && this.metricRef.current.mainCurve)) {
       return
     }
+    console.log('calculating pos')
```
Comment (haritonasty) [inline src/ducks/SANCharts/Charts.js]: don't forget to delete logging

Comment (DmitriVanGuard) [reply src/ducks/SANCharts/Charts.js]: Yep, thanks for reminding

**Pull request merged**